modules/shadertools/src/utils/webgl-info.js

```diff
@@ -32,11 +32,13 @@ Object.keys(WEBGL_FEATURES).forEach(key => {
 
 export {FEATURES};
 
-function isWebGL2(gl) {
+function isWebGL2(glAlias) {
   return Boolean(
-    gl &&
-      ((typeof WebGL2RenderingContext !== 'undefined' && gl instanceof WebGL2RenderingContext) ||
-        gl.TEXTURE_BINDING_3D === GL_TEXTURE_BINDING_3D)
+    glAlias &&
+      ((typeof WebGL2RenderingContext !== 'undefined' &&
```

Comment:
Nit: could be worth splitting into two expressions to make it easier to read.

Reply:
Why is this function different from the others?

Reply:
The difference is 'checking if context is of type is `WebGL2RenderingContext`'. This requires access to `global` class (belongs to `webgl` module), not all modules this method defined depend on `webgl` module, more details in the #1048

Reply:
There is indeed some special processing of types to make headless-gl types available on the global object, but that shouldn't affect this code (i.e. the browser will automatically put WebGL types on the global `window` object, and we do check that the type is defined before using it, so we should be safe.

I propose this implementation in all places:
```
  const isRealWebGL2Context = typeof WebGL2RenderingContext !== 'undefined' && glAlias instanceof WebGL2RenderingContext;
  const isWebGL2ContextLike = glAlias && (glAlias.TEXTURE_BINDING_3D === GL_TEXTURE_BINDING_3D);
  return isRealWebGL2Context || isWebGL2ContextLike;
```